test_scripts/API/SDL_Passenger_Mode/004_OnDriverDistraction_ignored_flow.lua

```diff
@@ -0,0 +1,42 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0119-SDL-passenger-mode.md
+--
+-- Description:
+-- In case:
+-- 1) By policy OnDriverDistractions allowed for (FULL, LIMITED, BACKGROUND, NONE) HMILevel
+-- 2) HMI sends invalid "OnDriverDistractions" notification to SDL:
+--  a) invalid data type
+--  b) empty value
+-- SDL does:
+-- 1) Send OnDriverDistraction notification to mobile with "lockScreenDismissalEnabled" parameter and all mandatories
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/SDL_Passenger_Mode/commonPassengerMode')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local invalidValue = {
+	"Invalid data type",  -- invalid data type
```

Comment:
@HSavynetska Please use the same indentation across all scripts. 
E.g. currently in `003` 2 spaces are used, however in `004` - 1 tab
I suggest to use 2 spaces everywhere.
In order to not miss this in the future please tune up IDE (Sublime) accordingly.